static/js/protein/chart.ts

```diff
@@ -447,7 +572,7 @@ export function drawProteinInteractionChart(
       window.location.href = `${PROTEIN_REDIRECT}${proteinId}`;
     })
     .on("mouseover", mouseover)
-    .on("mousemove", () => mousemove(TOOL_TIP_LEFT_POSITION, 510))
+    .on("mousemove", () => mousemove(TOOL_TIP_LEFT_POSITION, 650))
```

Comment:
Can you try to calculate the topPosition or in mousemove, calculate the y without topPosition. You don't want to have to update this value for all the charts every time there is an update to chart dimensions. 

there are a few different MouseEvent properties that you can try out.
documentation on MouseEvent properties: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
documentation on the diff between a few of the properties: https://stackoverflow.com/questions/6073505/what-is-the-difference-between-screenx-y-clientx-y-and-pagex-y

Reply:
That's a great idea! Used d3.event.pageX and d3.event.pageY instead!